### ISSUE TITLE
publish v0.0.1

### ISSUE THREAD

**Issue by Peyton-Spencer** [omniaura/agentflow-vscode#1]
https://code.visualstudio.com/api/working-with-extensions/publishing-extension
https://portal.azure.com/#home
https://learn.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=Windows

**Comment by Peyton-Spencer** [comment]
## Publish v0.0.1 — Dependency Map

After reviewing the project, here's what needs to happen before v0.0.1 can ship:

### Blockers (must-fix)
1. **#3 — package.json missing required fields** — `publisher` field is required by `vsce`. Also need icon, license field, and keywords for a proper listing.
2. **Azure DevOps publisher account** — need a publisher ID registered at https://marketplace.visualstudio.com/manage

### High priority (should-fix before publish)
3. **#2 — TextMate grammar missing conditional syntax** — the grammar only covers `.title` and `<!var>`, but agentflow now has conditionals (`<?...>`), end tags (`</...>`), else blocks, type annotations, comparison operators, and dot-notation paths. Publishing with incomplete highlighting would be a poor first impression.

### Medium priority (nice-to-have before publish)
4. **#4 — README has placeholder text** — marketplace listing pulls from README, so placeholders would look unprofessional.

### Suggested publish checklist
- [ ] Register publisher on Azure DevOps marketplace
- [ ] Fix #3 (package.json fields)
- [ ] Fix #2 (grammar coverage)
- [ ] Fix #4 (README cleanup)
- [ ] Run `vsce package` and test the .vsix locally
- [ ] Run `vsce publish`
